### PR TITLE
Done

### PR DESCRIPTION
**Pull request by jonpstone**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -251,18 +251,6 @@ <h4 class="student-name">Andy Touch</h4>
...
-              <div class="view-profile-div">
-                <h3 class="view-profile-text">View Profile</h3>
-              </div>
-              <div class="card-text-container">
-                <h4 class="student-name">John Carlo Malto</h4>
-                <p class="student-location">Quezon City, Philippines</p>
-              </div>
-            </a>
-          </div>
-
```
Comment (mendelB) [inline index.html]: Hi @jonpstone,
It seems as if you're changing someone elses data here which is problematic for the merge. Can you please fix this? (If you need help with this you can post on the Ask a Question bar or ping me on slack @mendel)

File: students/jonathan-stone.html [hunk 2]
```diff
@@ -58,10 +56,10 @@ <h2 class="profile-location">Scottsdale, AZ</h2>
               <h3>Experience</h3>
             </div>
             <div class="description-holder">
-              <h4>Factory X</h4>
-              <h5>Fellowship</h5>
-              <h4>GoFundVeterans</h4>
-              <h5>co-founder</h5>
```
Comment (mendelB) [inline students/jonathan-stone.html]: This file is problematic as well, you're deleting someone else's file and substituting it with your own.

Comment (jonpstone) [inline students/jonathan-stone.html]: Hi, so sorry, I'm not sure how I ended up doing that. Okay, I'll got back and try to alter it.
Regards,

J.P. Stone
Jonathan Stone | 4640 Hedgcoxe Road | Apartment 1312 | Plano | Texas | 75024 | Skype: stoney9007 | Telephone: +1 469.352.3235 

    On Wednesday, April 6, 2016 4:13 PM, Mendel Blesofsky <notifications@github.com> wrote:
 

 In students/jonathan-stone.html:> @@ -58,10 +56,10 @@ <h2 class="profile-location">Scottsdale, AZ</h2>
>                <h3>Experience</h3>
>              </div>
>              <div class="description-holder">
> -              <h4>Factory X</h4>
> -              <h5>Fellowship</h5>
> -              <h4>GoFundVeterans</h4>
> -              <h5>co-founder</h5>
This file is problematic as well, you're deleting someone else's file and substituting it with your own.—

Comment (jonpstone) [inline students/jonathan-stone.html]: Hi, okay so I resurrected my poor fellow students who I mistakenly butchered with my amateur coding. I also made some changes in the stylesheet. Hope this is what's needed.

**Comment by mendelB** [comment]
Looking good @jonpstone I'll merge this in. Thanks!

**Pull request merged**